01 SFC Hello VueJS/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "sample-vue-js",
+  "version": "1.0.0",
+  "description": "Sample working SFC VueJs with Typescript and Webpack",
+  "main": "webpack.config.js",
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --mode development --progress --hot --open",
```

Comment:
these scripts are not wrote on readme, `cross-env` is not installed, do we need add these scripts? or using like sample before?

Reply:
I'm sorry, my mistake. Fixed.